src/constants/permissions.ts

```diff
@@ -195,8 +255,16 @@ values.SYSCONSOLE_ANCILLARY_PERMISSIONS = {
         values.READ_JOBS,
         values.DOWNLOAD_COMPLIANCE_EXPORT_RESULT,
     ],
-    [values.SYSCONSOLE_READ_ENVIRONMENT]: [
```

Comment:
Tagged you in on one of my PR's that gets rid of these SYSCONSOLE Ancillary on the frontend. Will help avoid having to manage this.